go.mod

```diff
@@ -1,15 +1,17 @@
 module github.com/stackmon/otc-status-dashboard
 
-go 1.22
+go 1.22.0
+
+toolchain go1.23.2
```

Comment:
Please use go version 1.22 exactly. Because firstly we need to check, that 1.23 can be possible to use.

Reply:
Just remove this line :)

Reply:
set toolchain go1.22.9